__tests__/actions/user.js

```diff
@@ -6,6 +6,30 @@ import thunk from 'redux-thunk';
 import * as actions from '../../src/actions/user';
 
 describe('Actions: User', () => {
+  describe('action creators', () => {
+    it('creates correct SAVE_ACCESS_TOKEN object', () => {
+      const token = 'token';
+      const expected = {
+        type: actions.SAVE_ACCESS_TOKEN,
+        payload: token,
+      };
+      expect(actions.saveAccessToken(token)).toEqual(expected);
+    });
+    it('creates correct SAVE_ACCESS_TOKEN object', () => {
```

Comment:
This appears to be an exact duplicate of the prior test. Would be good to remove it.